sympy/functions/combinatorial/numbers.py

```diff
@@ -59,38 +60,36 @@ class carmichael(Function):
 
     Certain cryptographic algorithms make use of big prime numbers.
     However, checking whether a big number is prime is not so easy.
-    Randomized prime number checking tests exist that offer a high degree of confidence of
-    accurate determination at low cost, such as the Fermat test.
+    Randomized prime number checking tests exist that offer a high degree of
+    confidence of accurate determination at low cost, such as the Fermat test.
 
-    Let 'a' be a random number between 2 and n - 1, where n is the number whose primality we are testing.
-    Then, n is probably prime if it satisfies the modular arithmetic congruence relation :
+    Let 'a' be a random number between $2$ and $n - 1$, where $n$ is the
+    number whose primality we are testing. Then, $n$ is probably prime if it
+    satisfies the modular arithmetic congruence relation:
+
+    .. math :: a^{n-1} = 1 \pmod{n}
 
-    a^(n-1) = 1(mod n).
     (where mod refers to the modulo operation)
 
     If a number passes the Fermat test several times, then it is prime with a
     high probability.
 
-    Unfortunately, certain composite numbers (non-primes) still pass the Fermat test
-    with every number smaller than themselves.
+    Unfortunately, certain composite numbers (non-primes) still pass the Fermat
+    test with every number smaller than themselves.
     These numbers are called Carmichael numbers.
 
-    A Carmichael number will pass a Fermat primality test to every base b relatively prime to the number,
-    even though it is not actually prime. This makes tests based on Fermat's Little Theorem less effective than
-    strong probable prime tests such as the Baillie-PSW primality test and the Miller-Rabin primality test.
-    mr functions given in sympy/sympy/ntheory/primetest.py will produce wrong results for each and every
-    carmichael number.
```

Comment:
This sentence didn't make sense to me so I removed it. In what sense "wrong results"? (It may have been fixed since added though.)